.github/workflows/pr-test.yml

```diff
@@ -85,3 +85,43 @@ jobs:
       - name: Run integration tests
         run: |
           pipenv run pytest integration_tests -k 'not api_key'
+
+  performance-tests:
+    strategy:
+      fail-fast: true
+      matrix:
+        python: ["3.7", "3.8", "3.9", "3.10"]
+    env:
+      working-directory: ./performance_tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python }}
+      - uses: actions/setup-node@v2
+      - uses: azure/setup-helm@v1
+      - uses: dschep/install-pipenv-action@v1
+        with:
+          # version 2021.11.5, 2021.11.5.post0 do not work
```

Comment:
@tronxd Why does it not work? How old is the version we're using?

Reply:
there was an issue with both of the version, I pinned it on the weekend. They published a new one, probably worth a try. Remove `with` block -> run it -> works then great, if not put it back

Reply:
@gruebel Worked, thanks!